setup.cfg

```diff
@@ -15,6 +15,9 @@ install_requires =
   tensorflow
   networkx
   ray[tune,rllib]
+  torch
+  torchaudio
+  torchvision
```

Comment:
Do we need these? Just wondering what's provided here versus the base package

Reply:
Hmm, good question, probably not.  I do this by default because that's what they have in their docs.

Reply:
great